scrapers/DEU/DE-BY/index.js

```diff
@@ -0,0 +1,56 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as datetime from '../../../lib/datetime.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+// const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  country: 'DEU',
+  state: 'DE-BY', // ISO 3166 notation
+  url: 'https://raw.githubusercontent.com/jgehrcke/covid-19-germany-gae/master/data.csv',
+  type: 'json',
+  timeseries: true,
+  curators: [
+    {
+      name: 'Dr. Jan-Philip Gehrcke',
+      email: 'jgehrcke@googlemail.com'
+    }
+  ],
+  sources: [
+    {
+      name: 'Robert Koch-Institut, Bavarian health ministry',
+      description: 'Fresh data obtained from Bavarian health ministry by ZEIT ONLINE'
```

Comment:
Does there happen to be a URL for this? It would be good to add it.

Reply:
Sure. Thanks. Can I add a `url` property here?

I don't want to take too much credit here, but I think a good URL would be https://github.com/jgehrcke/covid-19-germany-gae -- in the README I tried to explain rather in-depth which sources are used for what.

Reply:
Yes, you absolutely can add a URL there, if it details where the data is from, it's relevant!